### DISCUSSION TITLE
插件“内容”页面里的“电子书元数据”的几个选项都是什么意思啊？

### DISCUSSION THREAD

**Discussion by GoTop** [bookfere/Ebook-Translator-Calibre-Plugin#259] [Q&A]
![WX20240330-083307@2x](https://github.com/bookfere/Ebook-Translator-Calibre-Plugin/assets/1784319/221f6fd9-5678-40fc-bcbf-a27810b59f62)

**Comment by bookfere** [comment]
启用这些选项会修改 .opf 文件中的元数据。

* __语言标记__：勾选后书名会附加目标语言，如“__Test 测试__”会变成“__Test 测试 [Simplified Chinese]__”
* __语言代码__：勾选后会将元数据中的语言代码替换成目标语言代码，如原来是 __en__ 会被替换成  __zh__
* __主题__：填写的主题会添加到元数据中，每行对应一个 `<subject>` 元素

**Reply by GoTop** [reply]
主题 就是标签的意思吧，自动将 主题 里的元素添加到书籍的标签里

这个功能很自动为翻译生成的双语书，标注“已翻译”、“双语”，这样就不用我自己手动添加了

**Reply by bookfere** [reply]
这个和翻译没关系。已存在的 `<subject>` 会被自动翻译，在设置中填写的是新往里添加的，和手动编辑元数据往里添加新主题一样。

**Comment by GoTop** [comment]
那“主题”具体是指什么，在书籍的“编辑元数据”没有看到关于“主题”的选项

**Reply by bookfere** [reply]
具体定义可参考 Open Packaging Format (OPF) 规范中的 [`2.2.3: <subject> </subject>`](https://idpf.org/epub/20/spec/OPF_2.0.1_draft.htm#Section2.2.3) 小节。
